lib/email/reader.rb

```diff
@@ -0,0 +1,51 @@
+class Email::Reader
+  EMAIL_REGEX = /(.*)--inline.html/
+  PATH_TO_EMAILS = '/emails'
```

Comment:
Freeze mutable objects assigned to constants.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.